pkg/resources/serviceports/service_port.go

```diff
@@ -11,6 +11,12 @@ var appProtocolHTTP = "http"
 func GetDefaultPorts() *[]corev1.ServicePort {
 
 	ports := []corev1.ServicePort{
+		{
+			Name:       "tcp-jgroup",
```

Comment:
Could you change the port name to `jgroups-tcp` or just `jgroups`?

Reply:
I can change it to jgroups and set the appProtocol to tcp. I used this name because of this documentation https://istio.io/latest/docs/ops/configuration/traffic-management/protocol-selection/

Reply:
> I can change it to jgroups and set the appProtocol to tcp. I used this name because of this documentation https://istio.io/latest/docs/ops/configuration/traffic-management/protocol-selection/

Your proposal LGTM +1

Reply:
I change it. It's ready to review again.